### PR TITLE
Refactor s2i Tests for ci-operator

### PR DESCRIPTION
**Pull request by adambkaplan**
* Isolate integration tests that don't require Docker
* Add OWNERS file
* Add make commands to run integration tests
* Bump golang to 1.12

### CODE REVIEW

File: test/integration/docker/integration_test.go [hunk 1]
```diff
@@ -0,0 +1,737 @@
+// +build integration
+
+package docker
```
Comment (adambkaplan) [inline test/integration/docker/integration_test.go]: This was moved from test/integration, plus I deleted all `*Dockerfile*` tests

File: test/integration/dockerfile/dockerfile_test.go [hunk 2]
```diff
@@ -1,590 +1,22 @@
 // +build integration
 
-package integration
+package dockerfile
```
Comment (adambkaplan) [inline test/integration/dockerfile/dockerfile_test.go]: This wasn't a rename - I moved all of the Dockerfile test code to a separate package.

**Comment by adambkaplan** [comment]
[test]

**Comment by adambkaplan** [comment]
[test]

**Comment by bparees** [comment]
@adambkaplan i'm unsubscribing from this, please re-tag me when you have passing tests.

**Comment by adambkaplan** [comment]
@bparees this is now ready for review

**Comment by bparees** [comment]
[merge]

**Pull request merged**